src/test/java/hudson/remoting/RegExpBenchmark.java

```diff
@@ -32,9 +32,9 @@
 import java.util.jar.JarEntry;
 import java.util.jar.JarFile;
 import java.util.regex.Pattern;
-import org.junit.jupiter.api.Disabled;
+import org.junit.Ignore;
 
-@Disabled("This is not a test just a benchmark and is here for ease of running")
+@Ignore("This is not a test just a benchmark and is here for ease of running")
```

Comment:
Consistently fails for me locally; not sure how it even passes on CI. Something about JRE vs. JDK?